docs/src/pages/Popper/6-manual-trigger.vue

```diff
@@ -0,0 +1,82 @@
+<title>Manual Triggers</title>
+
+<template>
+  <div class="container">
+    <FdButton ref="target" styling="light">Popper Target Reference</FdButton>
+    <FdPopper
+      ref="popper"
+      :target="target"
+      placement="bottom"
+      triggers="manual"
+      arrow-element=".popper-arrow"
+      :outside-dismiss="false"
+    >
+      <div class="popover">
+        <div class="popper-arrow"></div>
+        <div>
+          <b>Manually trigger the popper</b>
+          <br />This can be used for the use case of autocomplete input
+        </div>
+      </div>
+    </FdPopper>
+
+    <FdButton @click="toggle()">Manual trigger Popper hide/show</FdButton>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+export default Vue.extend({
+  data() {
+    return {
+      popper: null as Vue | Element | Vue[] | Element[] | null,
+      target: null as Vue | Element | Vue[] | Element[] | null
+    };
+  },
+  mounted() {
```

Comment:
Alternative way of writing it:

```
async mounted() {
   await this.$nextTick();
   this.create();
}
```

Reply:
Cool. got it. will push this in next commit